Vector.cpp

```diff
@@ -63,7 +91,7 @@ namespace Micromouse
 			assert( _x < NUM_NODES_W );
 			assert( _y < NUM_NODES_H );
 
-			// no Node should exist at two odd coordinates
+			// no Pos should have a position at two odd coordinates
 			assert( ( _x % 2 != 1 ) && ( _y % 2 != 1 ) );
```

Comment:
This was already an error with this code but this will fail whenever Pos has an odd coordinate, not when both of its coordinates are odd.